<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>NEURONAS</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" rowspan="8">
         <widget class="QPlainTextEdit" name="mallaMostrar"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>ID:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLineEdit" name="datoId"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Voltaje:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QLineEdit" name="datoVoltaje"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Posición X:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLineEdit" name="datoPosicionX"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Posición Y:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QLineEdit" name="datoPosicionY"/>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Red:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QLineEdit" name="datoRed"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Green:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2" colspan="2">
         <widget class="QLineEdit" name="datoGreen"/>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Blue:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2" colspan="2">
         <widget class="QLineEdit" name="datoBlue"/>
        </item>
        <item row="8" column="4">
         <widget class="QPushButton" name="mostrarNeuronas">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QPushButton" name="insertarInicio">
          <property name="text">
           <string>Insertar Inicio</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2" colspan="2">
         <widget class="QPushButton" name="insertarFinal">
          <property name="text">
           <string>Insertar Final</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Mostrar</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>521</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonMostrardelatabla">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Mostrar de la tabla</string>
        </property>
       </widget>
       <widget class="QPushButton" name="BotonBuscardelatabla">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Buscar por ID</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="idBusqueda">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>171</width>
          <height>31</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string> ID</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gráfica</string>
       </attribute>
       <widget class="QPushButton" name="dibujar">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Dibujar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="limpiar">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>220</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Limpiar</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>10</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
